### ISSUE TITLE
Logger Lock failed. Error (err_no=30)

### ISSUE THREAD

**Issue by dimitriovic** [NebuTech/NBMiner#823]
I have a mixed Nvidia rig: 4 fhr and 3 lhr. Work fine but not quite stable. Either Windows crach or Nbminer restarts after error after few minutes or 2-3 hours.
Few times it was a 1660ti incriminated with this error message the last time:
Error - Logger lock failed. Error - CUDA Error: unknown error (err_no=30)

- should I remove lock clock for this gpu or all the fhr gpu ?
- should I separate the fhr and lhr gpu ?

**Comment by Tom5051** [comment]
I find you get cuda error 30 if your memory clocks are too high. Take off 25mhz from the offending GPU until it becomes stable.

**Comment by dimitriovic** [comment]
> I find you get cuda error 30 if your memory clocks are too high. Take off 25mhz from the offending GPU until it becomes stable.

yeah ? tbh that's what I'm doing since the first crashes. I'm lowering the oc's by 10's mhz. and if you read the release notes, he says it clearly that we have to lower our memclock about 100mhz to 200mhz. So i lowered it little by little and the hashrate didn't move nearly at all. this may be the reason why it crashes, so I lowered all my oc by 100mhz, even the non lhr in a new bat file. now I'm waiting the next crash if any and see how it behaves this new oc. keep you updated.
thanks for reply btw, gave me confidence that it is indeed related to oc
